src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraPreviewControlRenderer.cs

```diff
@@ -1012,7 +1070,50 @@ private async Task StopCamera2Preview(ICameraPreviewStopParameters parameters)
                 StopBackgroundThread();
             }
 
-            cameraPreviewControl.SetStatus(CameraPreviewControl.CameraStatus.Stopped);
+            cameraPreviewControl.SetStatus(CameraStatus.Stopped);
+        }
+
+#pragma warning disable 1998
```

Comment:
Can we add a comment saying why we're disabling this warning?

Reply:
Yep